### PR TITLE
 Change static "ARK" to networkToken() in price tabs

### PR DESCRIPTION
**Pull request by geopsllc**
## Proposed changes

Change static "ARK" to networkToken() in price tabs. Essentially it pulls the token name from the API instead of using static text. This makes it easy to port the explorer to other chains.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/components/ContentHeader.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
         <span class="block md:inline-block">{{ height.toLocaleString() }}</span>
       </div>
       <div class="mr-2">
-        <span>ARK/{{name}}:</span>
+        <span>{{ networkToken() }}/{{name}}:</span>
```
Comment (dated) [inline src/components/ContentHeader.vue]:
```suggestion
        <span>{{ networkToken() }}/{{ name }}:</span>
```

**Review by dated — APPROVED** [review]

**Comment by ItsANameToo** [comment]
@faustbrian don't merge this yet, I don't think this is a change we want but will check properly tomorrow. The reason certain tables have no ID column is because they are not showing actual IDs but just an icon with a link to a details screen

**Comment by samharperpittam** [comment]
> @faustbrian don't merge this yet, I don't think this is a change we want but will check properly tomorrow. The reason certain tables have no ID column is because they are not showing actual IDs but just an icon with a link to a details screen.

Rok was referring to the ID column within the Transaction table that was previously removed.

**Comment by faustbrian** [comment]
https://github.com/ArkEcosystem/explorer.ark.io/pull/441

**Pull request merged**